From 35417c9613a62d5fdd2db05fc7d8995af47859e2 Mon Sep 17 00:00:00 2001
From: Wentao Guan <guanwentao@uniontech.com>
Date: Wed, 8 May 2024 18:11:37 +0800
Subject: [PATCH 028/100] net/stmmac: Add phytium old dwmac acpi_device_id

As Phytium ACPI Description Specification document v1.2 p13
Device HID said for old v1.0 Spec,add FTGM0001 for such as
some FT2000 device compat.

Signed-off-by: Wentao Guan <guanwentao@uniontech.com>
Ref: https://github.com/deepin-community/kernel-rolling/commit/71379b1f6b41a17fcae76e326eb433c7c9ab6918
---
 drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c b/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
index 6ecfa671b7f37..dd4171a39a4a9 100644
--- a/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
+++ b/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
@@ -198,6 +198,7 @@ MODULE_DEVICE_TABLE(of, phytium_dwmac_of_match);
 
 #ifdef CONFIG_ACPI
 static const struct acpi_device_id phytium_dwmac_acpi_ids[] = {
+	{ .id = "FTGM0001" }, // compat FT2000/4 id
 	{ .id = "PHYT0004" },
 	{ }
 };
-- 
2.46.0

